kafka/channel/pkg/dispatcher/dispatcher.go

```diff
@@ -139,26 +163,19 @@ func (c consumerMessageHandler) Handle(ctx context.Context, consumerMessage *sar
 
 	c.logger.Debug("Going to dispatch the message",
 		zap.String("topic", consumerMessage.Topic),
-		zap.Any("destination", c.sub.SubscriberURI),
-		zap.Any("reply", c.sub.ReplyURI),
-		zap.Any("delivery", c.sub.Delivery),
+		zap.String("subscription", c.sub.String()),
```

Comment:
oh, was that the reason of the issues ?

Reply:
no, but it was causing other issues in logging. The reason of issues was the previous `subscription` type used by the dispatcher